src/clients/c++/request.h

```diff
@@ -298,8 +298,8 @@ class InferContext {
     virtual Error SetShape(const std::vector<int64_t>& dims) = 0;
 
     /// Set tensor values for this input from a byte array. The array
-    /// is not copied and so it must not be modified or destroyed
-    /// until this input is no longer needed (that is until the Run()
+    /// is not copied and so  shared memory region must not be modified or
```

Comment:
Why does this mention shared memory? I don't think this comment needs to change.